pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/model/ComputeField.java

```diff
@@ -142,7 +141,6 @@ private Class<?> getJavaType() {
         case BOOLEAN:
           return Boolean.class;
         case DATE:
-          return Date.class;
```

Comment:
Good thing is primitive types support on compute is not released yet so no one could be using `DATE` + primitive types so we should be safe here.